docs/docs/understanding/workflows/state.md

```diff
@@ -56,7 +56,7 @@ async def step_two(self, ctx: Context, ev: StepTwoEvent) -> StopEvent:
     # do something with the data
     print("Data is ", await ctx.get("some_database"))
 
-    return StopEvent(result=await ctx.get("some_database")[1])
+    return StopEvent(result=await ctx.get("some_database"))
```

Comment:
```suggestion
    return StopEvent(result=(await ctx.get("some_database"))[1])
```

Reply:
But, do we really need to confuse a beginner level reader with indexing? I feel, It causes more confusion and issues (due to typos) in the StopEvent. Pls suggest.

Reply:
@sharathe100111 if the user gets confused by accessing a list, it's even more important that we show them how to access its items in an async fashion. Again, the example works nevertheless, but I believe in the fact examples should teach something. If you think parentheses are too visually confusing, we can do
```py
databases = await ctx.get("some_database")
return StopEvent(result=databases[1])
```

Reply:
yes, I agree. But, why do we really need indexing here?

- By using indexing it returns `2`, without any context of why it is returning only `2`. [`User is not specifically querying for second element in database`]
- If we don't use indexing, it returns actual data `[1, 2, 3]` as-is as database response. 

@masci, I'm ready to accept the changes. But requesting one last review on this change before we commit to it.